src/xdocs/checks/naming/membername.xml

```diff
@@ -26,40 +26,38 @@
             </tr>
             <tr>
               <td>applyToPackage</td>
-              <td>
-                Controls whether to apply the check to package-private member.
-              </td>
+              <td>Controls whether to apply the check to package-private members.</td>
               <td><a href="../../property_types.html#boolean">boolean</a></td>
               <td><code>true</code></td>
               <td>3.4</td>
             </tr>
             <tr>
               <td>applyToPrivate</td>
-              <td>Controls whether to apply the check to private member.</td>
+              <td>Controls whether to apply the check to private members.</td>
               <td><a href="../../property_types.html#boolean">boolean</a></td>
               <td><code>true</code></td>
               <td>3.4</td>
             </tr>
             <tr>
               <td>applyToProtected</td>
-              <td>Controls whether to apply the check to protected member.</td>
+              <td>Controls whether to apply the check to protected members.</td>
               <td><a href="../../property_types.html#boolean">boolean</a></td>
               <td><code>true</code></td>
               <td>3.4</td>
             </tr>
             <tr>
               <td>applyToPublic</td>
-              <td>Controls whether to apply the check to public member.</td>
+              <td>Controls whether to apply the check to public members.</td>
               <td><a href="../../property_types.html#boolean">boolean</a></td>
               <td><code>true</code></td>
               <td>3.4</td>
             </tr>
             <tr>
               <td>format</td>
-              <td>Specifies valid identifiers.</td>
+              <td>Sets the pattern to match valid identifiers.</td>
               <td><a href="../../property_types.html#Pattern">Pattern</a></td>
               <td><code>&quot;^[a-z][a-zA-Z0-9]*$&quot;</code></td>
-              <td>3.0</td>
+              <td>3.4</td>
```

Comment:
Please put hacks in place to avoid battle on `since` versions. There are a couple of these.

Reply:
done